Forge.TreeWalker.UnitTests/test/TreeWalkerUnitTests.cs

```diff
@@ -243,6 +243,20 @@ public void TestTreeWalkerSession_WalkTree_ActionHasDelay_ContinuationOnTimeout_
             Assert.AreEqual("TimeoutOnAction", actionResponse.Status, "Expected WalkTree to be successful because the TreeAction timed out but ContinuationOnTimeout flag was set.");
         }
 
+        [TestMethod]
+        public void TestTreeWalkerSession_WalkTree_ActionHasDelay_ContinuationOnRetryExhaustion_RetryPolicy_FixedCount()
+        {
+            // Initialize TreeWalkerSession with a schema containing Action with throw an excetion but RetryPolicy is fixedCount so should try.
+            this.TestInitialize(jsonSchema: ForgeSchemaHelper.ActionDelay_ContinuationOnRetryExhaustion_RetryPolicy_FixedCount);
+
+            // Test - Expect WalkTree to be successful because the TreeAction timed out but ContinuationOnRetryExhaustion flag was set.
```

Comment:
>because the TreeAction timed out but [](start = 55, length = 36)

nit: Clean up this comment a bit since we didn't hit a timeout here.

Reply:
Similarly clean up the matching Assert comments as well.

---
In reply to: [401968392](https://github.com/microsoft/Forge/pull/13#discussion_r401968392) [](ancestors = 401968392)

Reply:
TreeAction did not time out here. This is the RetryExhaustion case.

---
In reply to: [401968549](https://github.com/microsoft/Forge/pull/13#discussion_r401968549) [](ancestors = 401968549,401968392)